datm/cime_config/config_component.xml

```diff
@@ -94,6 +94,7 @@
       <value compset="_SLND">none</value>
       <value compset="_DLND">none</value>
       <value compset="_DATM%CPLHIST">cplhist</value>
+      <value compset="_DATM%MOSARTTEST">observed</value>
```

Comment:
Remove.

Reply:
It is done with https://github.com/ESCOMP/CDEPS/pull/2/commits/768fc9e6ba01f6bbfb5fb79de6269cc5762138a8